library/L2_HAL/sensors/environment/light/temt6000x01.hpp

```diff
@@ -13,36 +13,29 @@ class Temt6000x01 final : public LightSensor
 {
  public:
   /// @param adc The ADC peripheral used to read the sensor data.
-  /// @param adc_reference_voltage The reference voltage of the ADC. This value
-  ///                              will vary based on the chosen MCU.
   /// @param pull_down_resistance The resistance of the pull down resistor used
   ///                             in the circuit.
   explicit constexpr Temt6000x01(sjsu::Adc & adc,
-                                 units::voltage::volt_t adc_reference_voltage,
```

Comment:
It would be a good idea to keep this in the constructor, but in this case, the reference voltage wouldn't be for the ADC it would be for the supply voltage for the light sensor.